lib/src/spec/validationEngineValidateForm.spec.ts

```diff
@@ -42,23 +40,23 @@ describe('ValidationEngine Validate Form', () => {
 
             expect(formValidationResult.fieldErrors).to.have.length(1);
             expect(formValidationResult.fieldErrors[0].succeeded).to.be.false;
-            expect(formValidationResult.fieldErrors[0].key).to.be.equal('nameId');
+            expect(formValidationResult.fieldErrors[0].key).to.be.equal('fullname');
             expect(formValidationResult.fieldErrors[0].type).to.equal('REQUIRED');
             expect(formValidationResult.fieldErrors[0].errorMessage).to.equal('Mandatory field');
             done();
           });
       });
 
-    it('Spec #2 => should return a succeeded FormValidationResult with one fieldErrors equals { succeeded: true, key: "nameId", type: "REQUIRED", errorMessage: "" }' +
-      'When passing one validation rule with key equals "nameId" and viewModel equals { id: "1", fullname: "john" }', (done) => {
+    it('Spec #2 => should return a succeeded FormValidationResult with one fieldErrors equals ' +
```

Comment:
I was thinking about that... but remember that even if we use backticks messages may be displayed with same indentation they have in the text editor (at least we don't use 4-8 tabs wide)...